src/main/resources/db/migration/V1.17.0__Create_break_flag_table.sql

```diff
@@ -0,0 +1,7 @@
+use SeichiAssist;
+
+CREATE TABLE break_flags(
+    uuid CHAR(36) NOT NULL PRIMARY KEY,
+    flag_name VARCHAR(20) NOT NULL,
```

Comment:
```suggestion
    flag_name VARCHAR(20) NOT NULL,
```

事故を防ぐためにこのカラムは[`ENUM`](https://mariadb.com/kb/en/enum/)であるべきです。